google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/impl/http/ForwardProxyPoolSettings.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.googlecloud.bigquery.impl.http
```

Comment:
I'm not entirely sure, but this forward proxy mechanism is also shared code among other connectors is it?

Reply:
Yes that's right. Actually, I think my `BigQueryHttp` class should become a common `GoogleHttp` class that handles the proxy stuff, requests with OAuth, retries, etc.